package.json

```diff
@@ -15,11 +15,14 @@
         "editor",
         "tools",
         "template",
```

Comment:
should be named "templates" (with the s)

Reply:
see my comment above about folder name conflicts

Reply:
done